src/api/api.js

```diff
@@ -29,6 +29,15 @@ const usersAPI = {
   getProfile(userId) {
     return instance.get(`profile/${userId}`);
   },
+
+  getStatus(userId) {
+    return instance.get(`profile/status/${userId}`);
```

Comment:
Consider creating a const object "ROUTES" at the constants.js level and storing ALL the related routes there.
`const ROUTES = { PROFILE_STATUS: 'profile/status', ...};`

Go through your application and rewrite the code to support references from ROUTES object.